src/router/trees.js

```diff
@@ -13,29 +13,40 @@ export const routes = [
     props: (route) => ({ uuid: route.params.uuid }),
     component: () => import(/* webpackChunkName:"chunk-builder" */ '@/views/FetchFlow.vue'),
   },
-  {
-    path: '/trees/create', /* no-op */
-  },
   {
     path: '/trees/:id/interaction-designer/:mode',
+    alias: '/flows/:id/interaction-designer/:mode',
+    name: 'flow-canvas',
     props: (route) => ({ id: route.params.id, mode: route.params.mode }),
     component: () => import(/* webpackChunkName:"chunk-builder" */ '@/views/InteractionDesigner.vue'),
-  }, {
```

Comment:
Can you explain me why we deleted these ?

Reply:
I have removed the old routes and re-created them using [nested routing](https://router.vuejs.org/guide/essentials/nested-routes.html). We had some hard-coded routes which would get complicated as we add new routes. Hence, I chose to create nested routing which also provides accurate structure.

Reply:
thanks for explaining, I just wanted to make sure it has been deleted on purpose